### PR TITLE
[ADD][11.0] Adding module project_agile_analytic

### PR DESCRIPTION
**Pull request by pnajman-modoolar**
Hello,

Module ``project_agile_analytic`` brings analytic entries for ``project.task`` and based on those entries we should be able to generate cumulative flow diagrams and other agile reports.

Kind Regards,
Petar

### CODE REVIEW

File: project_agile_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_agile_analytic/README.rst]: see https://github.com/OCA/web/pull/881#pullrequestreview-99094143

File: project_agile_analytic/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "summary": "This module provides analytics for project tasks",
+    "category": "Project",
+    "version": "11.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Odoo Community Association (OCA), Modoolar",
+    "website": "https://www.modoolar.com/",
+    "depends": ["project_agile"],
+    "data": [
+
+        # security
```
Comment (elicoidal) [inline project_agile_analytic/__manifest__.py]: remove these comments

File: project_agile_analytic/models/analytic_line.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        comodel_name='res.company',
+        string='Company',
+        required=True,
+        index=True,
+    )
+
+    @api.multi
+    def _compute_duration(self):
+        for record in self:
+            s = fields.Datetime.from_string(record.start_date)
```
Comment (elicoidal) [inline project_agile_analytic/models/analytic_line.py]: name of the variables could be improved

File: project_agile_analytic/models/analytic_line.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            e = record.end_date and \
+                fields.Datetime.from_string(record.end_date) or datetime.now()
+
+            delta = e - s
+            hours = delta.total_seconds() / 36000
```
Comment (elicoidal) [inline project_agile_analytic/models/analytic_line.py]: 36000 or 3600?

File: project_agile_analytic/views/project_agile_analytic_views.xml [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+
+    <record id="view_task_form2_analytics" model="ir.ui.view">
+        <field name="name">view_task_form2_analytics</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <div name="button_box">
+                <field name="analytic_enabled" invisible="1"/>
+                <button name="%(project_agile_task_analytic_line_action)d" type="action" groups="project.group_project_manager"
```
Comment (elicoidal) [inline project_agile_analytic/views/project_agile_analytic_views.xml]: CAn you improve the readability of the XML line?

File: project_agile_analytic/views/project_agile_analytic_views.xml [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+    <!--Add field to project form-->
+    <record id="edit_project_analytics" model="ir.ui.view">
+        <field name="name">edit.project.analytics</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="priority" eval="20"/>
+        <field name="arch" type="xml">
+            <div name="button_box">
+                <field name="analytic_enabled" invisible="1"/>
+                <button name="%(project_agile_analytic_line_action)d" type="action" groups="project.group_project_manager"
```
Comment (elicoidal) [inline project_agile_analytic/views/project_agile_analytic_views.xml]: here too?

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details.
No UT or demo data

File: project_agile_analytic/README.rst (hunk 1 above)
Comment (pnajman-modoolar) [reply project_agile_analytic/README.rst]: Fixed

File: project_agile_analytic/__manifest__.py (hunk 2 above)
Comment (pnajman-modoolar) [reply project_agile_analytic/__manifest__.py]: Done

File: project_agile_analytic/models/analytic_line.py (hunk 3 above)
Comment (pnajman-modoolar) [reply project_agile_analytic/models/analytic_line.py]: Done

File: project_agile_analytic/models/analytic_line.py (hunk 4 above)
Comment (pnajman-modoolar) [reply project_agile_analytic/models/analytic_line.py]: Fixed. Thanks :)

File: project_agile_analytic/views/project_agile_analytic_views.xml (hunk 5 above)
Comment (pnajman-modoolar) [reply project_agile_analytic/views/project_agile_analytic_views.xml]: Hope it's ok now ...

**Pull request closed without merging**